apps/rush-lib/src/logic/deploy/DeployManager.ts

```diff
@@ -16,7 +16,9 @@ import {
   JsonSchema,
   IPackageJson,
   AlreadyExistsBehavior,
-  InternalError
+  InternalError,
+  NewlineKind,
+  Text
 } from "@rushstack/node-core-library";
```

Comment:
Yeah, I was having trouble keeping the quotes straight because we use `"` at work.  Once this is in, I'm thinking we just merge a PR that runs prettier on the `master` branch and normalizes everything. We need to get the commit hook set up.